src/views/clusteroverview/overview/components/SettingsTab/LiveMigrationTab/utils/utils.ts

```diff
@@ -0,0 +1,46 @@
+import HyperConvergedModel from '@kubevirt-ui/kubevirt-api/console/models/HyperConvergedModel';
+import { V1MigrationConfiguration } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { k8sPatch, K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+
+export type HyperConverged = K8sResourceCommon & {
+  spec: {
+    liveMigrationConfig: V1MigrationConfiguration;
+  };
+};
+
+export type HyperConvergedList = K8sResourceCommon & {
+  items: HyperConverged[];
+};
+
+export const updateLiveMigrationConfig = (
+  hyperConverged: HyperConverged,
+  value: number | string | null,
+  name: string,
+) =>
+  k8sPatch({
+    model: HyperConvergedModel,
+    resource: hyperConverged,
+    data: [
+      {
+        op: 'replace',
+        path: `/spec/liveMigrationConfig/${name}`,
+        value,
+      },
+    ],
+  });
+
+export const getHyperConvergedObject = (hyperConverged): HyperConverged => {
+  if (hyperConverged?.items) return hyperConverged?.items?.[0];
+  if (Array.isArray(hyperConverged)) return hyperConverged?.[0];
+  return hyperConverged;
+};
```

Comment:
now this should be something like 
```suggestion
export const getHyperConvergedObject = (hyperConverged: V1KubeVirt[]): V1KubeVirt => {
  return hyperConverged?.[0];
};
```

Reply:
i wish... :)